configs/config.toml.SAMPLE

```diff
@@ -108,4 +108,9 @@
 # The port on which the `/diagnostics` endpoint will be available can be
 # customized below.
 # [Diagnostics]
-	# Port = 8081
\ No newline at end of file
+	# Port = 8081
+
+# Uncomment to enable application-specific extensions.
+# [ApplicationExtensions.TBTC]
```

Comment:
Thoughts on making it just `Extensions.TBTC` for simplicity? Looking at the code, we have `InitializeExtensions`, `WithTBTCExtensions` so it seems like a natural thing to do.